common/test/acceptance/tests/lms/test_bookmarks.py

```diff
@@ -195,13 +258,34 @@ def test_unreachable_bookmark(self):
         When I click on deleted bookmark
         Then I should navigated to 404 page
         """
-        self._bookmarks_blocks(self.course_fixture.get_nested_xblocks(category="problem"))
-
+        self._test_setup()
+        self._bookmark_units(2)
         self._delete_section(0)
 
-        self.bookmarks.click_bookmarks_button()
-        self.assertTrue(self.bookmarks.results_present())
-        self.assertEqual(self.bookmarks.count(), 2)
+        self.bookmarks_page.click_bookmarks_button()
+        self.assertTrue(self.bookmarks_page.results_present())
+        self.assertEqual(self.bookmarks_page.count(), 2)
 
-        self.bookmarks.click_bookmark(0)
+        self.bookmarks_page.click_bookmarked_unit(0)
         self.assertTrue(is_404_page(self.browser))
+
+    def test_page_size_limit(self):
```

Comment:
So this tests server pagination? Could you also add a test for the 500 client-side pagination limit? I understand that these tests are for the list page, not the bookmark button page, so you could create a separate story for those tests.